### PR TITLE
ConvexYieldWrapper: removed unused named returns

### PR DESCRIPTION
**Pull request by iamsahu**
Fix for:
- https://github.com/code-423n4/2022-01-yield-findings/issues/60

### CODE REVIEW

File: contracts/oracles/convex/Cvx3CrvOracle.sol [hunk 1]
```diff
@@ -79,7 +79,7 @@ contract Cvx3CrvOracle is IOracle, AccessControl {
         bytes32 base,
         bytes32 quote,
         uint256 baseAmount
-    ) external view virtual override returns (uint256 quoteAmount, uint256 updateTime) {
+    ) external view virtual override returns (uint256, uint256) {
         return _peek(base.b6(), quote.b6(), baseAmount);
```
Comment (alcueca) [inline contracts/oracles/convex/Cvx3CrvOracle.sol:83]: The following allows you to keep the named returns, which are nice for code parsers such as Etherscan's.
```
) external view virtual override returns (uint256 quoteAmount, uint256 updateTime) {
    (quoteAmount, updateTime) = _peek(base.b6(), quote.b6(), baseAmount);
```

File: contracts/oracles/convex/Cvx3CrvOracle.sol [hunk 2]
```diff
@@ -96,7 +96,7 @@ contract Cvx3CrvOracle is IOracle, AccessControl {
         bytes32 base,
         bytes32 quote,
         uint256 baseAmount
-    ) external virtual override returns (uint256 quoteAmount, uint256 updateTime) {
+    ) external virtual override returns (uint256, uint256) {
```
Comment (alcueca) [inline contracts/oracles/convex/Cvx3CrvOracle.sol:99]: Same as above

**Review by alcueca — CHANGES_REQUESTED** [review]
We can keep the named returns, just by assigning to them.

**Review by devtooligan — APPROVED** [review]
code lgtm.  confirmed we save 3 gas per redundant return ommited

Screen Shot 2022-02-14 at 8 23 13 AM

**Review by alcueca — APPROVED** [review]
:+1:

**Pull request merged**